docker/privx-api/Dockerfile

```diff
@@ -5,5 +5,5 @@ COPY requirements.txt .
 
 RUN apk --update add --no-cache --virtual .build-dependencies python3-dev build-base wget git \
   && pip install --no-cache-dir -r requirements.txt \
+  && pip install git+https://github.com/SSHcom/privx-sdk-for-python.git@v34.0.0#egg=privx_api \
```

Comment:
Why is this needed? You do it in the Pipfile

Reply:
Correct, it's not needed here. Removed it.